### PR TITLE
[tutorial] fixed missing RIR scripts file.

### PR DESCRIPTION
**Pull request by XuesongYang**
# What does this PR do ?

missing get_openslr_rir_data.py file in RIR section.

**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Jenkins CI
To run Jenkins, a NeMo User with write access must comment `jenkins` on the PR.

# Before your PR is "Ready for review"
**Pre checks**:
- [x] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [x] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

**Comment by XuesongYang** [comment]
jenkins

File: tutorials/asr/Online_Noise_Augmentation.ipynb [hunk 1]
```diff
@@ -752,8 +752,9 @@
             "source": [
                 "# This is where the rir data will be downloaded.\n",
                 "# Change this if you don't want the data to be extracted in the current directory.\n",
+                "!wget https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/scripts/dataset_processing/get_openslr_rir_data.py\n",
                 "rir_data_path = '.'\n",
-                "!python ../../scripts/dataset_processing/get_openslr_rir_data.py --data_root {rir_data_path}\n",
+                "!python get_openslr_rir_data.py --data_root {rir_data_path}\n",
```
Comment (stevehuang52) [inline tutorials/asr/Online_Noise_Augmentation.ipynb:757]: why not reuse the script under nemo, such as `!python ../../../scripts/dataset_processing/get_openslr_rir_data.py --data_root {rir_data_path}`?

Comment (XuesongYang) [reply tutorials/asr/Online_Noise_Augmentation.ipynb:757]: hmm...i did not find `scripts` folder there.

Comment (stevehuang52) [reply tutorials/asr/Online_Noise_Augmentation.ipynb:757]: my bad, the original `../../` should lead to NeMo's root folder, which means `../../scripts/dataset_processing/get_openslr_rir_data.py` is a valid file

Comment (XuesongYang) [reply tutorials/asr/Online_Noise_Augmentation.ipynb:757]: thanks! but again, it still `failed` when running through google colab. It's not a big problem though. Let's wait next QA's report then.

```
python3: can't open file '/content/../../scripts/dataset_processing/get_openslr_rir_data.py': [Errno 2] No such file or directory
head: cannot open './processed/rir.json' for reading: No such file or directory
```

Comment (stevehuang52) [reply tutorials/asr/Online_Noise_Augmentation.ipynb:757]: maybe some env vars are wrong on collab, but the fix should work

**Review by stevehuang52 — APPROVED** [review]
not sure if it's necessary, but the change won't break existing code

**Pull request merged**